lib/correios_sigep/logistic_reverse/collect_number_actions/process_invalid_request.rb

```diff
@@ -23,9 +20,25 @@ def errors
             Models::CorreiosResponseCodes::INEXISTENT_ZIPCODE => Models::Errors::InexistentZipcode,
             Models::CorreiosResponseCodes::UNAVAILABLE_HOUSE_COLLECT => Models::Errors::UnavailableHouseCollect,
             Models::CorreiosResponseCodes::COLLECT_NOT_ANSWERED_FOR_THE_ZIPCODE => Models::Errors::CollectNotAnsweredForTheZipcode,
-            Models::CorreiosResponseCodes::NOT_CONFIGURED_CLIENT => Models::Errors::NotConfiguredClient
+            Models::CorreiosResponseCodes::NOT_CONFIGURED_CLIENT => Models::Errors::NotConfiguredClient,
+            Models::CorreiosResponseCodes::INVALID_CONTRACT => Models::Errors::InvalidContract,
+            Models::CorreiosResponseCodes::INVALID_DECLARED_VALUE => Models::Errors::InvalidDeclaredValue
           }
         end
+
+        def error_code
+          @response.search('//cod_erro').text.to_i
```

Comment:
`new_error_code` check if the result of the first search is empty before. 
On this, it is secure to go without the same checking?

Reply:
Good idea! Afaik, this always comes in the response but.. I'll update the PR!